eigensystem.py

```diff
@@ -282,9 +289,11 @@ def single_file(input_filename):
     fout["n_tilde"] = n_tilde
     fout["ir (base 0)"] = ir_list
     fout["ir (base 1)"] = [ir+1 for ir in ir_list]
+    fout["radii (cm)"]= rgrid
```

Comment:
Could we name this the same as the variable in the file? That would make it easier to interpret, since we would not have to remember the different names a particular quantity has.